templates/cluster/hetzner-standalone-cp/values.yaml

```diff
@@ -0,0 +1,59 @@
+# Cluster parameters
```

Comment:
Remove all values unless absolutely required. The actual values should be set from the ClusterDeployment and not hardcoded into the values.yaml here which will become a part of the released helm chart.